### ISSUE TITLE
Pointer events on Safari

### ISSUE THREAD

**Issue by VillePakarinenPosti** [timmywil/panzoom#631]
### Describe the bug

I was fiddling around with the panzoom demo at: https://timmywil.com/panzoom/demo/ and noticed that using 3-fingers within panzoom element causes quite a lot of layout-shift.

### Your environment

- Version of panzoom: Demo version.
- Browser and browser version: iOS 16.0.3. iOS on Safari

### Expected behavior

I'm not sure what should happen while panning with three fingers but iOS opens undo menu. Eg: https://developer.apple.com/forums/thread/714425

### Steps to reproduce

1. Open: https://timmywil.com/panzoom/demo/
2. Choose any of the demos.
3. Touch screen with 3 fingers
4. Demo tiger will flicker and might disappear from the view

**Comment by timmywil** [comment]
Good catch. There are a few possible fixes and I'm not sure the best one.

**Comment by howfar9595** [comment]
hello, will this be resolved？Make the drag bounded
